config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -31,6 +31,8 @@ presubmits:
         - --extract=local
         - --flush-mem-after-build=true
         - --gcp-nodes=100
+        # TODO(k/k#94098): Remove the line below once we get enough data.
+        - --gcp-master-size=n1-standard-8
```

Comment:
Let's do the same for post submit in master branch.

Reply:
Done